community/values/src/main/java/org/neo4j/values/storable/PointValue.java

```diff
@@ -255,4 +261,268 @@ public boolean withinRange( PointValue lower, boolean includeLower, PointValue u
         }
         return true;
     }
+
+    public static PointValue fromMap( MapValue map )
+    {
+        AnyValue[] fields = new Value[PointValueField.__MAX_VALUE__.ordinal()];
```

Comment:
It feels like you could as well use `PointValueField.values().length` and get rid of this `__MAX_VALUE__` altogether.